test/helpers/smartcards.ts

```diff
@@ -13,19 +15,185 @@ const contest0 = election.contests[0] as CandidateContest
+        partyId: '0',
+      },
+    ],
+    senator: [{ id: 'weiford', name: 'Dennis Weiford', partyId: '0' }],
+    'representative-district-6': [
+      { id: 'plunkard', name: 'Brad Plunkard', partyId: '0' },
+    ],
+    governor: [{ id: 'franz', name: 'Charlene Franz', partyId: '0' }],
+    'lieutenant-governor': [
+      { id: 'norberg', name: 'Chris Norberg', partyId: '0' },
+    ],
+    'secretary-of-state': [
+      { id: 'shamsi', name: 'Laila Shamsi', partyId: '0' },
+    ],
+    'state-senator-district-31': [
+      { id: 'shiplett', name: 'Edward Shiplett', partyId: '3' },
+    ],
+    'state-assembly-district-54': [
+      { id: 'solis', name: 'Andrea Solis', partyId: '0' },
+    ],
+    'county-commissioners': [
+      { id: 'argent', name: 'Camille Argent', partyId: '0' },
+    ],
+    'county-registrar-of-wills': [
+      { id: 'ramachandrani', name: 'Rhadka Ramachandrani', partyId: '6' },
+    ],
+    'city-mayor': [{ id: 'white', name: 'Orville White', partyId: '1' }],
+    'city-council': [{ id: 'eagle', name: 'Harvey Eagle', partyId: '0' }],
+    'judicial-robert-demergue': 'yes',
+    'judicial-elmer-hull': 'yes',
+    'question-a': 'yes',
+    'question-b': 'yes',
+    'question-c': 'yes',
+    'proposition-1': 'yes',
+    'measure-101': 'yes',
+  }
+)
+
+export const sampleVotes2: VotesDict = vote(
+  getContests({
+    ballotStyle: election.ballotStyles[0],
+    election: election,
+  }),
+  {
+    '102': 'no',
+    president: [
+      {
+        id: 'cramer-vuocolo',
+        name: 'Adam Cramer and Greg Vuocolo',
+        partyId: '1',
+      },
+    ],
+    senator: [{ id: 'garriss', name: 'Lloyd Garriss', partyId: '1' }],
+    'representative-district-6': [
+      { id: 'reeder', name: 'Bruce Reeder', partyId: '1' },
+    ],
+    governor: [{ id: 'harris', name: 'Gerald Harris', partyId: '1' }],
+    'lieutenant-governor': [
+      { id: 'parks', name: 'Anthony Parks', partyId: '1' },
+    ],
+    'secretary-of-state': [
+      { id: 'talarico', name: 'Marty Talarico', partyId: '1' },
+    ],
+    'state-senator-district-31': [
+      { id: 'shiplett', name: 'Edward Shiplett', partyId: '3' },
+    ],
+    'state-assembly-district-54': [
+      { id: 'keller', name: 'Amos Keller', partyId: '1' },
+    ],
+    'county-commissioners': [
+      {
+        id: 'witherspoonsmithson',
+        name: 'Chloe Witherspoon-Smithson',
+        partyId: '0',
+      },
+    ],
+    'county-registrar-of-wills': [
+      { id: 'write-in__HOT DOG', name: 'HOT DOG', isWriteIn: true },
+    ],
+    'city-mayor': [{ id: 'seldon', name: 'Gregory Seldon', partyId: '2' }],
+    'city-council': [{ id: 'rupp', name: 'Randall Rupp', partyId: '0' }],
+    'judicial-robert-demergue': 'no',
+    'judicial-elmer-hull': 'no',
+    'question-a': 'no',
+    'question-b': 'no',
+    'question-c': 'no',
+    'proposition-1': 'no',
+    'measure-101': 'no',
+  }
+)
+
+export const sampleVotes3: VotesDict = vote(
```

Comment:
Also, you can ditch the `VotesDict` here and elsewhere since that's already the return type of `vote`.

```suggestion
export const sampleVotes3 = vote(
```